android/src/newarch/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -199,6 +202,11 @@ public void setLackPermissionToDownloadMessage(RNCWebView view, @Nullable String
         mRNCWebViewManagerImpl.setLackPermissionToDownloadMessage(value);
     }
 
+    @ReactProp(name = "hasOnOpenWindowEvent")
+    public void setHasOnOpenWindowEvent(RNCWebView view, boolean hasEvent) {
```

Comment:
I'm not an expert on the new architecture yet either, but I notice that *most* of the other props here have the `@Override` attribute...

Reply:
Fixed in: https://github.com/react-native-webview/react-native-webview/compare/5a835821371ba93f5a5ac120c88097fc3c918ff4..610aee3309dfd69739c28a5bb4579b91015d0394